docs/core/reference/changelog.md

```diff
@@ -4,12 +4,26 @@ This changelog covers updates to cored, the Chain Core server binary.
 
 For updates to subpackages, see below:
 
-- [Mac installer](https://github.com/chain/chain/blob/1.2-stable/installer/mac/CHANGELOG.md)
-- [Windows installer](https://github.com/chain/chain/blob/1.2-stable/installer/windows/CHANGELOG.md)
+- [Mac installer](https://github.com/chain/chain/blob/1.2-stable/desktop/mac/CHANGELOG.md)
+- [Windows installer](https://github.com/chain/chain/blob/1.2-stable/desktop/windows/CHANGELOG.md)
 - [Java SDK](https://github.com/chain/chain/blob/1.2-stable/sdk/java/CHANGELOG.md)
 - [Node.js SDK](https://github.com/chain/chain/blob/1.2-stable/sdk/node/CHANGELOG.md)
 - [Ruby SDK](https://github.com/chain/chain/blob/1.2-stable/sdk/ruby/CHANGELOG.md)
 
+<a name="1.2.1"></a>
+## 1.2.1 (June 13, 2017)
+
+### Major changes
+
+TLS version 1.2 is now required for all HTTPS connections ([#1314](https://github.com/chain/chain/pull/1314)).
```

Comment:
This is really just a concern for clients. I didn't really look at the Java-related documentation very closely, so I wasn't sure if the settings were about setting TLS for servers, or if they were for enabling support for clients as well.